content/en/building/tasks/tasks-overview.md

```diff
@@ -32,6 +32,34 @@ Each task has an icon on the left side which indicates which type of task it is.
 Tasks are listed in order of due date. Tasks that are past due will appear at the top of the list. CHWs should strive to complete tasks before they are overdue. Many programs add targets to track task completion and timeliness.
 
 
+<br clear="all">
+
+## Task prioritization
+
+{{< callout >}}
+Available from 4.21.0.
+{{< /callout >}}
+
+Prioritization enhances task sorting to help CHWs focus on urgent tasks first. Previously, tasks were sorted only by due date, requiring users to manually scan for urgent cases. This feature introduces a priority score that is based on clinical or operational urgency such as danger signs, referral follow-ups, or high-risk conditions.
+
+Tasks are now sorted using the following logic:
+- Tasks with higher priority scores appear first.
+- When scores tie, tasks are sorted by due date.
+- Tasks without or invalid priority scores appear last.
+
```

Comment:
I think it's good to be very specific and straight forward about backwards compatibility for this. 

```suggestion
{{< callout >}}
Older configurations that do not implement the new priority score will continue sorting tasks by due date. 
{{< /callout >}}
```